cms/conf.py

```diff
@@ -202,6 +204,13 @@ def __init__(self):
         # Attempt to load a config file.
         self._load(paths)
 
+        if bool(self.telegram_bot_token) ^ bool(self.telegram_bot_chat_id):
+            raise ConfigError("Both telegram_bot_token and telegram_bot_chat_id "
+                              "should be set or left null")
+        if self.telegram_bot_chat_id:
+            if type(self.telegram_bot_chat_id) != int:
+                raise ConfigError("telegram_bot_chat_id should be an integer")
```

Comment:
These checks are probably better moved together with the other Telegram code to improve locality. This should be easier with the below suggestion.